website/content/docs/configuration/storage/raft.mdx

```diff
@@ -135,6 +136,16 @@ delay) mode. The maximum allowed value is 10.
     default to a value larger than the Integrated Storage default of 1MB, then you will 
     need to make the same change in Vault's Integrated Storage config.
 
+- `max_mount_and_namespace_table_entry_size` `(integer)`- <EnterpriseAlert
+  inline /> Overrides `max_entry_size` to set a different limit for the specific
+  KV entries that contain mount tables, auth tables and namespace configuration
```

Comment:
I could imagine `KV entries` being confusing for a reader, thinking that this might be referring to the kv secrets backend. Maybe `storage entries` would be clearer?